extensions/wolfram-alpha/src/index.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  ActionPanel,
+  Detail,
+  Form,
+  SubmitFormAction,
+  useNavigation,
+  showToast,
+  ToastStyle,
+  getPreferenceValues,
+  environment,
+  OpenInBrowserAction,
+  Icon,
+} from "@raycast/api";
+import fetch from "node-fetch";
+import { useState } from "react";
+
+interface Preferences {
+  appId: string;
+  units?: string;
+}
+
+export default function Command() {
+  const preferences: Preferences = getPreferenceValues();
+  const [loading, setLoading] = useState(false);
+  const [query, setQuery] = useState("");
+  const { push } = useNavigation();
+
+  // @ts-expect-error theme isn't there yet
+  const theme = environment.theme || null;
+
+  const onSubmit = async () => {
+    if (!query) {
+      showToast(ToastStyle.Failure, "Please enter something to query");
+      return;
+    }
+    setLoading(true);
+    try {
+      const params = new URLSearchParams({
+        appid: preferences.appId,
+        i: query,
+        width: `${474 * 2}`,
+        units: preferences.units || "metric",
+        fontsize: `${14 * 2}`,
+      });
+      if (theme) {
+        params.append("background", "transparent");
+        params.append("foreground", theme === "light" ? "black" : "white");
+      }
+      const res = await fetch(`http://api.wolframalpha.com/v1/simple?${params.toString()}`);
```

Comment:
```suggestion
      const res = await fetch(`https://api.wolframalpha.com/v1/simple?${params.toString()}`);
```
Without `https` it fails

Reply:
Oh they changed it on Friday! It used to work on Friday morning, not anymore haha